src/lsp/lp_doc.ml

```diff
@@ -153,20 +164,27 @@ let dummy_loc =
 
 let check_text ~doc =
   let uri, version = doc.uri, doc.version in
+  let root =
+    match doc.root with
+    | Some(ss) -> ss
+    | None ->
+      raise(Error.fatal_no_pos "Root signature is missing
+      probably because new_doc has raised exception")
```

Comment:
Just to know: will this error message be printed somewhere in the editor?

Reply:
Not displayed in the editor but will be written in the /tmp/lambdapi_lsp_log.txt file